synapse/lib/layer.py

```diff
@@ -1174,28 +1178,34 @@ async def _layrV2toV3(self):
                 sode['valu'] = (valu, stortype)
                 continue
 
-            if flag == 1:
```

Comment:
Due to the continue within each block, the "else if" logic is redundant and actually removes clarity to have the elif *and* the continue.  Additionally, it causes the need for the else: block, whose logic would otherwise not need an additional indent level.  Unless i'm missing something this is a purely cosmetic change that moves *away* from elegance.